Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -232,8 +232,7 @@ public async Task Push(Image x, string name, string? tag, string baseName, Actio
 
         if (!putResponse2.IsSuccessStatusCode)
         {
-            string jsonResponse = await putResponse2.Content.ReadAsStringAsync();
-            throw new ContainerHttpException("Registry push failed.", putResponse2.RequestMessage?.RequestUri?.ToString(), jsonResponse);
+            throw new ContainerHttpException("Registry push failed.", putResponse2.RequestMessage?.RequestUri?.ToString(), jsonString);
```

Comment:
the `jsonstring` here is the same as the `putResponse` - is that intended? I'd expect the content from `putResponse2` to go here.

Reply:
yeah, both responses use the same `manifestUploadContent`. in theory the only difference is the URI's they put to.

Reply:
Thanks for the confirmation 👍